01-main/packages/ookla-speedtest

```diff
@@ -0,0 +1,7 @@
+DEFVER=1
+ASC_KEY_URL="https://packagecloud.io/ookla/speedtest-cli/gpgkey"
+APT_LIST_NAME="speedtest"
```

Comment:
only needed where one repository hosts several apps, in which case we need this to align all to one list and key.  If you don't need it then omit it.